src/subsystems.jl

```diff
@@ -132,6 +132,35 @@ function get_participating_subsystems(
     return [k for (k, v) in data.subsystems if uuid in v]
 end
 
+"""
+Return true if the component is participating in any subsystems.
+"""
+function is_participating_in_subsystem(
```

Comment:
Question: should we use the term `assigned` instead of `participating`?

Reply:
I prefer `assigned`